controller.go

```diff
@@ -48,6 +49,15 @@ const (
 
 	// The amount of time  between node status checks
 	nodeMonitorPeriod = 30 * time.Second
+
+	// Guest cluster environment variable names.
+	clusterAPIEnv = "GUEST_CLUSTER_API"
+	clusterIDEnv  = "GUEST_CLUSTER_ID"
```

Comment:
Now we have env vars and this:

```
	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
	flag.StringVar(&master, "master", "", "master url")
	flag.BoolVar(&help, "help", false, "Print usage and exit")
```

I think we should have unified way to pass config, I personally prefer flags. We can discuss that, but that's what we had so far and let's not introduce that in this PR.